app/models/about.rb

```diff
@@ -1,6 +1,22 @@
 # frozen_string_literal: true
 
 class About
+  cattr_reader :plugin_stat_groups, :displayed_plugin_stat_groups
+
+  def self.add_plugin_stat_group(prefix, show_in_ui: false, &block)
+    if !@@displayed_plugin_stat_groups.include?(prefix) && show_in_ui
+      @@displayed_plugin_stat_groups << prefix
+    end
+    @@plugin_stat_groups[prefix] = { block: block }
```

Comment:
```suggestion
    @@plugin_stat_groups[prefix] = block
```

The hash is kind of redundant now.